analytics/views.py

```diff
@@ -1024,7 +1024,7 @@ def fix_rows(i: int,
 @require_server_admin
 @has_request_variables
 def get_activity(request: HttpRequest) -> HttpResponse:
-    duration_content, realm_minutes = user_activity_intervals()  # type: Tuple[mark_safe, Dict[str, float]]
+    duration_content, realm_minutes = user_activity_intervals()
```

Comment:
@andersk  you forgot to add the variable annotation here.

Reply:
It’s not needed; `user_activity_intervals()` has a correctly annotated return type.